### PR TITLE
Syllabifier Integration into TeXShop

### PR DESCRIPTION
**Pull request by rpspringuel**
This is not ready for merge, but I wanted to put it out there for comment at this point.

### CODE REVIEW

File: tips/texshopsyllabifier.html [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+--applescript direct
+
+set docname to #DOCUMENTNAME#
+
+tell application "TeXShop"
+	set input to content of selection of document docname
+end tell
+
+try
+	set output to do shell script ("/bin/echo \"" & input & "\" | ~/scripts/syllabify")
```
Comment (eroux) [inline tips/texshopsyllabifier.html]: there are a few invalid html chars here, `&` should be remplaced by `&amp;`

**Comment by eroux** [comment]
this is an excellent idea! It doesn't look very simple but I guess with the TeXShop system it can't really be...

**Comment by rpspringuel** [comment]
It's actually not so much TeXShop that is the problem, but the restrictions that Apple in general and AppleScript in particular places on GUI apps.  These are for the purpose of "security" but they're damned inconvenient when it comes to customizing apps.

**Comment by rpspringuel** [comment]
This now incorporates #72 into it and thus should be merged only after that one.

**Comment by rpspringuel** [comment]
I also want to look into doing something similar in TeXworks, but that will have to wait for the summer.  It's exam crunch time now and I don't have the spare time to research its macro extension language.

**Comment by rpspringuel** [comment]
Okay, it's been over a year since I touched this.  Scripting in TeXworks turns out to be fairly different from scripting in TeXShop so I'm probably not going to adapt this to TeXworks anytime soon.  As a result, I'm going to say this should be merged as is.

**Pull request merged**